### PR TITLE
Update CSS content in HTML/CSS Lesson 1

### PR DESCRIPTION
**Pull request by bonboh**
## What does this change?

Module: HTML/CSS
Week(s): Lesson 1

## Description

This PR updates HTML/CSS lesson 1 with CSS fundamentals, ahead of Scotland teaching the module next month.

These fundamentals were added as they are important concepts that make CSS less magical to understand, and ensures that all students become familiar with this material. Having this information in the syllabus means students can use it as their main reference when reviewing concepts. This is particularly important early on when searching the web will possibly lead them to content that is at the intermediate/advanced level.

More specificaly, this PR:

- Moves the "HTML/CSS Project" section to the beginning of the lesson
- Adds "What is HTML?" and "What is CSS?" sections to  introduce the HTML and CSS sections
- Adds "CSS Anatomy", "Common CSS properties", "Block and Inline Boxes", and "CSS Inheritance, Cascade and Specificity" sections
- Updates the "CSS Selectors" and "CSS Box Model" sections
- Moves "CSS Units" from lesson 2 to lesson 1 since it is a fundamental part of CSS
- Makes introduced keywords bold for learning emphasis
- Moves the links in the "Resources" section to more relevant sections
- Puts all exercises in admonitions to make them visually distinct from course content
- Adds tip admonitions to provide opinionated best practises

Lesson 1: https://github.com/CodeYourFuture/syllabus/blob/f690de6c8c2fc160be23215849cc5c1a74188e69/docs/html-css/week-1/lesson.md

### CODE REVIEW

**Comment by bonboh** [comment]
@ChrisOwen101 I've made some more changes to HTML/CSS lesson 1. Can you have a look please?

File: docs/html-css/week-2/lesson.md [hunk 1]
```diff
@@ -31,105 +31,6 @@ See how much variety there is in [viewport sizes](https://decadecity.net/blog/20
 
 As a group, let's brainstorm as many devices as we can think of which might access the websites we build.
 
-## Units of Measurement
```
Comment (ChrisOwen101) [inline docs/html-css/week-2/lesson.md:34]: What you're thinking about removing the unit's of measurement here?

**Review by ChrisOwen101 — COMMENTED** [review]
Really sensible suggestions in here - thanks @bonboh !

**Comment by ChrisOwen101** [comment]
@bonboh - Thanks! This has now been merged.

**Pull request merged**